client/debian/control

```diff
@@ -41,8 +47,8 @@ Description: Library for querying Ubuntu hardware certification status - Rust so
 Package: hwctl
 Architecture: amd64
 Multi-Arch: foreign
-Depends: ${misc:Depends}, ${shlibs:Depends}
-Description: Command-line tool for checking Ubuntu hardware certification status - Rust source code
+Depends: kmod, lsb-release, ${misc:Depends}, ${shlibs:Depends}
+Description: CLI tool for checking Ubuntu hardware certification status - Rust source code
```

Comment:
Has been shortened to fix "synopsis too long" lintian error

Reply:
Perhaps drop the acronym. It reads well as-is